.github/workflows/tests_linux.yml

```diff
@@ -150,3 +150,71 @@ jobs:
           pytest tests/
           pl-device-test --device lightning.kokkos --skip-ops --shots=20000
           pl-device-test --device lightning.kokkos --skip-ops --shots=None
+  cpptests_threads:
```

Comment:
Hi @mlxd , Another request to add `THREADS` the first sentence Line 39 of `README.rst`. Thanks

Reply:
We still have this note
```
Note that THREADS
backend is not recommended since `Kokkos <https://github.com/kokkos/kokkos-core-wiki/blob/17f08a6483937c26e14ec3c93a2aa40e4ce081ce/docs/source/ProgrammingGuide/Initialization.md?plain=1#L67>`_ does not guarantee its safety.
```
So we could add it last with `"THREADS" (not recommended)` for the lazy readers.

Reply:
Added